cli/jobs/single-step/r/accidents/src/aml.R

```diff
@@ -0,0 +1,110 @@
+library(mlflow)
```

Comment:
should we rename this file to something more descriptive? like azureml_utils.R

also, can you add some comments at the top of the file that describe what this utility file is for?

Reply:
Renamed the file. Please validate the content and provide feedback.